databases/week3/meal_sharing.sql

```diff
@@ -1,33 +1,461 @@
+);
+
+INSERT INTO `meal` (
+	`title`,
+	`description`,
+	`location`,
+	`when`,
+	`max_reservations`,
+	`price`,
+	`created_date`
+) VALUES (
+	"Vegan brunch",
+	"Oatmeal grød with almond milk",
+	"Vesterbro",
+	"2019-12-02",
+	4,
+	99,
+	"2019-11-25"
+);
+
+INSERT INTO `meal` (
+	`title`,
+	`description`,
+	`location`,
+	`when`,
+	`max_reservations`,
+	`price`,
+	`created_date`
+) VALUES (
+	"Danish brunch",
+	"Sausages, omlette and pancakes",
+	"Østerbro",
+	"2019-12-04",
+	4,
+	120,
+	"2019-11-25"
+);
+
+INSERT INTO `reservation` (
+	`number_of_guests`,
+	`meal_id`,
+	`created_date`
+) VALUES (
+	2,
+	2,
+	"2019-11-27"
+);
+
+INSERT INTO `reservation` (
+	`number_of_guests`,
+	`meal_id`,
+	`created_date`
+) VALUES (
+	2,
+	3,
+	"2019-11-27"
+);
+
+INSERT INTO `reservation` (
+	`number_of_guests`,
+	`meal_id`,
+	`created_date`
+) VALUES (
+	4,
+	1,
+	"2019-11-25"
+);
+
+INSERT INTO `reservation` (
+	`number_of_guests`,
+	`meal_id`,
+	`created_date`
+) VALUES (
+	2,
+	2,
+	NOW()
+);
+
+INSERT INTO `reservation` (
+	`number_of_guests`,
+	`meal_id`,
+	`created_date`
+) VALUES (
+	4,
+	2,
+	"2019-12-01"
+);
+
+INSERT INTO `reservation` (
+	`number_of_guests`,
+	`meal_id`,
+	`created_date`
+) VALUES (
+	4,
+	3,
+	"2019-12-01"
+);
+
+INSERT INTO `review` (
+	`title`,
+	`description`,
+	`meal_id`,
+	`stars`,
+	`created_date`
+) VALUES (
+	"Expensive",
+	"Taste was ok but they charge too much for an oat meal",
+	3,
+	2,
+	NOW()
+);
+
+INSERT INTO `review` (
+	`title`,
+	`description`,
+	`meal_id`,
+	`stars`,
+	`created_date`
+) VALUES (
+	"Nice",
+	"We liked the taste of everything",
+	2,
+	4,
+	NOW()
+);
+
+INSERT INTO `review` (
+	`title`,
+	`description`,
+	`meal_id`,
+	`stars`,
+	`created_date`
+) VALUES (
+	"Amazing",
+	"The best brunch I have ever had",
+	1,
+	5,
+	NOW()
+);
+
+INSERT INTO `review` (
+	`title`,
+	`description`,
+	`meal_id`,
+	`stars`,
+	`created_date`
+) VALUES (
+	"Average",
+	"It was alright but maybe I had bigger expectations and it couldn't fulfill them.",
+	2,
+	3,
+	NOW()
+);
+
+-- [FUNCTIONALITY]
+-- Get meals that has a price smaller than a specific price fx 90
+SELECT * FROM meal
+WHERE price < 100;
+
+-- Get meals that still has available reservations
+SELECT * FROM meal
+WHERE `when` > "2019-11-27";
+
+-- Get meals that partially match a title. 
+-- Rød grød med will match the meal with the title Rød grød med fløde
+SELECT * FROM meal
+WHERE `title` LIKE "%vegan%";
+
+-- Get meals that has been created between two dates
+SELECT * FROM meal
+WHERE created_date between "2019-11-26" and CURDATE()
+ORDER BY created_date;
+
+-- Get only specific number of meals fx return only 5 meals
+SELECT * FROM meal
+WHERE created_date between "2019-11-26" and CURDATE()
+ORDER BY created_date
+LIMIT 5;
+
+-- Get the meals that have good reviews
+SELECT meal.title, meal.description, meal.location, meal.when, meal.max_reservations, meal.price, review.stars
+FROM meal
+JOIN review ON meal.id = review.meal_id
+WHERE review.stars >= 3;
```

Comment:
This is great. Although it will very likely return duplicate rows, so one thing you could do to avoid would be to average the number of review stars and group by meal.id